data-collections-exercises.md

```diff
@@ -15,83 +15,91 @@ Access the second value in `planeteers`.
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+planeteers.push("Heart")
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+planeteers.delete("Captain Planet")
 ```
 
 Combine `planeteers` with `rangers = ["Red", "Blue", "Pink", "Yellow", "Black"]` and save the result in a variable called `heroes`.
 
 ```rb
-# Your answer here
+heroes = rangers.combination(planeteers)
```

Comment:
combination method takes a number not an array and has a different usage if we say `rangers.combination(3).to_a` it will give us 3 combinations for the array 
```
[["Red", "Blue", "Pink"], ["Red", "Blue", "Yellow"], ["Red", "Blue", "Black"], ["Red", "Pink", "Yellow"], ["Red", "Pink", "Black"], ["Red", "Yellow", "Black"], ["Blue", "Pink", "Yellow"], ["Blue", "Pink", "Black"], ["Blue", "Yellow", "Black"], ["Pink", "Yellow", "Black"]] 
```

the solution for the problem is to concatenate them using the '+' operator `heroes=planeteers+rangers`